mmseg/models/backbones/mit.py

```diff
@@ -0,0 +1,420 @@
+import math
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import (ConvModule, Linear, build_activation_layer,
+                      build_norm_layer, constant_init, normal_init,
+                      trunc_normal_init)
+from mmcv.cnn.bricks import DropPath
+from mmcv.runner import BaseModule, ModuleList, load_checkpoint
+from torch.nn.modules.utils import _pair as to_2tuple
+
+from ...utils import get_root_logger
+from ..builder import BACKBONES
+
+
+class DWConv(BaseModule):
+
+    def __init__(self, dim=768):
+        super().__init__()
+        self.dwconv = ConvModule(
+            in_channels=dim,
+            out_channels=dim,
+            kernel_size=3,
+            stride=1,
+            padding=(3 - 1) // 2,
+            bias=True,
+            groups=dim)
+
+    def forward(self, x, H, W):
+        B, _, C = x.shape
+        x = x.transpose(1, 2).reshape(B, C, H, W)
+        x = self.dwconv(x)
+        x = x.flatten(2).transpose(1, 2).contiguous()
+
+        return x
+
+
+class Mlp(BaseModule):
```

Comment:
Be careful that the variable or class name needs to be consistent with the paper.
Rename Mlp to MixFFN